drgn/helpers/linux/system.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+System Info
+--------------
+
+The ``drgn.helpers.linux.system`` module provides helpers for generic
+system information in the Linux kernel.
+"""
+
+from typing import List
+
+from drgn import Program
+
+__all__ = ("avenrun",)
+
+
+def avenrun(prog: Program) -> List[float]:
+    """
+    Return system load averaged over 1, 5 and 15 minutes as
+    list of three float values.
+
+    >>> avenrun(prog)
+    (2.34, 0.442, 1.33)
```

Comment:
With the current code, this would actually be:

```suggestion
    [2.34, 0.442, 1.33]
```

But seeing it this way, I think it would actually be better to return a `Tuple[float, float, float]`.

Reply:
Sure!